src/main/java/com/thealgorithms/maths/FindMaxRecursion.java

```diff
@@ -49,6 +30,6 @@ public static int max(int[] array, int low, int high) {
      * @return max value of {@code array}
      */
     public static int max(int[] array) {
-        return array.length == 1 ? array[0] : max(array, 0, array.length);
+        return array.length == 1 ? array[0] : max(array, 0, array.length - 1);
```

Comment:
```suggestion
        return max(array, 0, array.length - 1);
```

Reply:
I didnt change FindMax Recusion i just saw that error when i had a proper test. Seconf I would rather not use Math.max the performance shhould be equal and so the function works one itself without any other classes.

Reply:
My suggestion was about not using the `array.length == 1` (this reduces number of branches) and simply call your method `max` with proper arguments.